src/customers/account-dashboard-my-account.md

```diff
@@ -4,11 +4,12 @@ title: My Account
 
 The My Account section of the customer dashboard gives customers an overview of their information and activity related to your store.
 
-![]({% link images/images-ee/customer-account-dashboard-my-account.png %}){: .zoom}
-_My Account_{:.ee-only}
-
-## Recently Ordered Products
+The My Account page consist of following sections:
 
-The **Recent Orders** block is displayed in the sidebar and on the **My Account page** for customers who are logged in. The **Recently ordered** block appears after placing an order, it displays five products from the last purchase.
+- [Account Information] ({% link customers/account-dashboard-account-information.md %})
+- [Address Book] ({% link customers/account-dashboard-address-book.md %})
+- [My Recent Reviews] ({% link customers/account-dashboard-my-product-reviews.md %})
+- [Recent Orders] ({% link ccustomers/account-dashboard-my-orders.md %})
```

Comment:
```suggestion
- [Recent Orders] ({% link customers/account-dashboard-my-orders.md %})
```
Rake would have caught that.

Reply:
Fixed @dobooth , thank you 😊 
Could you please approve?